src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -201,6 +201,15 @@ def execute(self, reqStatus):
                     # then we still have pileup to be transferred, but with incorrect locations
                     msg = "Workflow: %s cannot proceed due to some PU misconfiguration. Check previous logs..."
                     self.logger.critical(msg, wflow.getName())
+                    # Send alert to prometheus
```

Comment:
Kenyi, I am inclined to say that the content of this alert should be encapsulated in a method/function, something similar to `notifyLargeData`. Such that this logic does not get too much polluted with the alert information. Maybe we could call it something like `def alertPUMisconfig`.